IOMC/EventVertexGenerators/python/VtxSmearedDesign_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+# Load GaussEvtVtx Generator and read parameters from GT (SimBeamSpotObjectRcd)
+from IOMC.EventVertexGenerators.VtxSmearedParameters_cfi import VtxSmearedCommon
+VtxSmeared = cms.EDProducer("GaussEvtVtxGenerator",
+    VtxSmearedCommon
+)
+VtxSmeared.readDB = True
+
+
+
```

Comment:
```suggestion
VtxSmeared = cms.EDProducer("GaussEvtVtxGenerator",
    src = cms.InputTag("generator", "unsmeared"),
    readDB = cms.bool(True)
)
```
```

Reply:
solved in last push, see https://github.com/cms-sw/cmssw/pull/43242#issuecomment-1805465244